src/powershell/install-chocolatey.ps1

```diff
@@ -0,0 +1,32 @@
+# Download and execute Chocolatey install script
+Set-ExecutionPolicy Bypass -Scope Process -Force
+[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072
```

Comment:
Is there a long flag option (or options) for the `-bor 3072` parameter?  I tried to find the documentation for this function, but came up empty.  If there is no long flag here (and maybe even if there is), perhaps a comment can help explain what's going on here.

Reply:
@dav3r resolved with [bbbf965](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/bbbf965b7583a5cb9b281bf9ee2fe901b8d6a534)